spec/helpers.rb

```diff
@@ -222,7 +222,7 @@ def create_and_review_incident
 
   def submit_to_state
     visit_status :state_submission
-    click_button 'FINAL SUBMISSION'
+    click_button 'SUBMISSION'
```

Comment:
Unrelated to this PR, but this (as well as the changes to `splash_page_spec` below) is needed so that tests pass regardless of branding.